CITATION.cff

```diff
@@ -0,0 +1,16 @@
+cff-version: 1.2.0
+title: Jenkins
+type: software
+repository-code: 'https://github.com/jenkinsci/jenkins'
+url: 'https://jenkins.io/'
```

Comment:
Yarn lint wants this to use double quotes instead of single quotes.  Is that acceptable to the consumers of the file?
```suggestion
repository-code: "https://github.com/jenkinsci/jenkins"
url: "https://jenkins.io/"
```

Reply:
Surely, that works fine. The underlying syntax is YAML, so that works either way 👍